features/gebruik_in_lopende_tekst.feature

```diff
@@ -1,189 +1,651 @@
+  - Wanneer een naamcomponent geen of een lege waarde heeft, wordt de overbodige spatie niet opgenomen: niet starten met een spatie, niet eindigen met een spatie, geen dubbele spatie, geen spatie na streepje
+
+  Abstract Scenario: gebruikInLopendeTekst bij geslachtsaanduiding <voorbeeld>
+    Gegeven het systeem heeft een persoon met de volgende gegevens
+    | naam                | waarde     |
+    | burgerservicenummer | 999992934  |
+    | geslachtsaanduiding | <geslacht> |
+    En de persoon heeft de volgende naam gegevens
+    | naam                                 | waarde      |
+    | voornamen (02.10)                    | Jo Rene     |
+    | adellijke titel of predikaat (02.20) |             |
+    | voorvoegsel (02.30)                  |             |
+    | geslachtsnaam (02.40)                | Groenen     |
+    | aanduidingNaamgebruik (61.10)        | E           |
+    Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 999992934                       |
+    | fields              | naam.gebruikinlopendetekst      |
+    Dan bevat de persoon met burgerservicenummer '999992934' de volgende 'naam' gegevens
+    | naam                  | waarde                  |
+    | gebruikinlopendetekst | <gebruikinlopendetekst> |
+
+    Voorbeelden:
+    | voorbeeld | geslacht | gebruikInLopendeTekst |
+    | man       | M        | de heer Groenen       |
+    | vrouw     | V        | mevrouw Groenen       |
+    | onbekend  | O        | J.R. Groenen          |
+
+  Scenario: gebruikInLopendeTekst van een alleenstaande persoon met aanduidingNaamgebruik "E"
+    Gegeven het systeem heeft een persoon met de volgende gegevens
+    | naam                | waarde    |
+    | burgerservicenummer | 999992934 |
+    | geslachtsaanduiding | V         |
+    En de persoon heeft de volgende naam gegevens
+    | naam                                 | waarde |
+    | adellijke titel of predikaat (02.20) |        |
+    | voorvoegsel (02.30)                  | in het |
+    | geslachtsnaam (02.40)                | Veld   |
+    | aanduidingNaamgebruik (61.10)        | E      |
+    En persoon heeft geen partner
+    Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 999992934                       |
+    | fields              | naam.gebruikinlopendetekst      |
+    Dan bevat de persoon met burgerservicenummer '999992934' de volgende 'naam' gegevens
+    | naam                  | waarde              |
+    | gebruikinlopendetekst | mevrouw In het Veld |
+
+  Abstract Scenario: naamsamenstelling van een persoon met een partner bij aanduidingNaamgebruik "<naamgebruik>"
```

Comment:
Wordt met naamsamenstelling gebruikInLopendeTekst bedoeld?
Is het nodig om gebruikInLopendeTekst op te nemen in de scenario titels?

Reply:
Laat maar, ik denk dat ik het snap nu ik de volgende scenario lees.

Reply:
het scenario gaat over de volgorde waarin de naamcomponenten worden opgenomen in gebruikInLopendeTekst plus over het verwijderen van dubbele spaties wanneer een naamcomponent (voorvoegsel) geen waarde heeft